### PR TITLE
Create 2022-11-11_mdanalysis.yaml

### PR DESCRIPTION
**Pull request by micaela-matta**
MDAnalysis project manager job ad

### CODE REVIEW

**Comment by stefanv** [comment]
@micaela-matta Thanks for the jobs post! Can you please fill out the contact URL field? If you can edit the post down a bit, that'd be great, but isn't strictly necessary.

File: jobs/2022-11-11_mdanalysis.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+  - Community engagement
+
+  - Industrial engagement and partnership development
+
+  - Communicating with users and developers
+
+  - Project management
+
+For more details, see the [full job description](https://www.mdanalysis.org/public/data/MDAnalysis_Manager_Advert.pdf).
```
Comment (stefanv) [inline jobs/2022-11-11_mdanalysis.yaml]:
```suggestion
```

**Comment by micaela-matta** [comment]
thanks for reviewing @stefanv!

**Comment by stefanv** [comment]
Thanks @micaela-matta, your post is now live on the jobs board and at permalink https://jobs.numfocus.org/job/2022-11-11_mdanalysis

**Pull request merged**